manifest.json

```diff
@@ -35,7 +35,8 @@
   ],
   "options_ui": {
     "page": "options.html",
-    "open_in_tab": true
+    "browser_style": true,
+    "chrome_style": true
```

Comment:
This property doesn't seem to have any effect. I couldn't find any documentation for it on MDN either.

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui